client/my-sites/themes/theme-options.js

```diff
@@ -89,14 +93,25 @@ const preview = {
 	label: i18n.translate( 'Live demo', {
 		comment: 'label for previewing the theme demo website'
 	} ),
-	hideForSite: ( state, siteId ) => isJetpackSite( state, siteId ),
+	action: ( themeId ) => requestPreview( themeId, 'wpcom' ),
 	hideForTheme: ( state, theme, siteId ) => isActive( state, theme.id, siteId )
 };
 
-const signup = {
-	label: i18n.translate( 'Pick this design', {
-		comment: 'when signing up for a WordPress.com account with a selected theme'
+const previewOnJetpack = {
```

Comment:
As discussed on Slack, and given that we've recently managed to merge JP/WP.com specific options for operations like `activate`, I'd love if we could stick to that pattern, i.e. one shared `preview` option only; possibly distinct `preview` actions (with one action that calls either, depending on whether it's a JP or WP.com site), similar to what the `activate` and `tryAndCustomize` actions do.